src/renderers/common/Backend.js

```diff
@@ -437,11 +445,33 @@ class Backend {
 	 *
 	 * @async
 	 * @abstract
-	 * @param {RenderContext} renderContext - The render context.
-	 * @param {String} type - The render context.
-	 * @return {Promise} A Promise that resolves when the time stamp has been computed.
+	 * @param {String} [type='render'] - The type of the time stamp.
+	 * @return {Promise<Number>} A Promise that resolves with the time stamp.
 	 */
-	async resolveTimestampAsync( /*renderContext, type*/ ) { }
+	async resolveAllTimestampsAsync( type = 'render' ) {
```

Comment:
Do you we need the `*All*` term in the name? How about `resolveTimestampsAsync()`?